ecommerce/enterprise_coupons/utils.py

```diff
@@ -0,0 +1,38 @@
+"""
+Helper methods for enterprise app.
+"""
+import logging
+
+from django.conf import settings
+from .backends import get_coupon_backend
+
+log = logging.getLogger(__name__)
+
+def get_enterprise_catalog_id_from_coupon(coupon_code, backend_alias='default'):
+    """Get catalog ID using configured backend."""
+    backend = get_coupon_backend(backend_alias)
+    return backend.get_catalog_id(coupon_code)
+    
+    
+def get_build_mfe_base_url(catalog_id):
+    """
+    Build the base URL for the ecommerce microfrontend receipt page with catalog parameter.
+
+    Args:
+        catalog_id (int): The catalog ID to include in the microfrontend URL.
+
+    Returns:
+        (str): The complete URL for the ecommerce microfrontend receipt page with the catalog parameter appended.
+        NoneType: Return None if MFE_REDIRECT_URL is not set in settings.
+    """
+    mfe_base_url = getattr(
+            settings,
+            'MFE_REDIRECT_URL',
+            None
+        )
+    if not mfe_base_url:
+        log.error("MFE_REDIRECT_URL is not set in settings.")
+        return None
+    
+    mfe_base_url = f"{mfe_base_url}?catalog={catalog_id}"
```

Comment:
mfe_base_url is only used once, so I think there is no need to define it. @daviidco